install/index.php

```diff
@@ -18,8 +18,8 @@
  */
 
 // The new version of THT we're installing
-define("NVER", "1.3");
-define("NVERCODE", 1010300);
+define("NVER", "1.4.0");
+define("NVERCODE", 1010400);
```

Comment:
Nope, thanks for the explanation. Moving to 1.4 can make sense given I've already established (years ago) what 1.3 was supposed to be. We'll just skip that release.